sys/net/application_layer/asymcute/asymcute.c

```diff
@@ -517,15 +517,20 @@ static void _on_unsuback(asymcute_con_t *con, const uint8_t *data, size_t len)
 static void _on_data(asymcute_con_t *con, size_t pkt_len, sock_udp_ep_t *remote)
 {
     size_t len;
-    size_t pos = _len_get(con->rxbuf, &len);
+    size_t pos;
+
+    if (pkt_len < 2) {
```

Comment:
I'm (still) in favour of moving this check to the beginning of the function, and thus leave the declaration and assignment of `pos` as it was.

Otherwise this is good to go

Reply:
please amend (or squash) directly, so we get this through quickly